docs/conda_environment.yml

```diff
@@ -0,0 +1,16 @@
+channels:
+  - conda-forge
+  - ncar
+dependencies:
```

Comment:
This file seems to have been copied from geocat-comp and has several unnecessary dependencies for GeoCAT-viz, which should be removed.

Reply:
You are absolutely correct on where this file came from, haha. I've modified it now.